ground/build.gradle

```diff
@@ -206,7 +206,9 @@ dependencies {
     implementation "com.google.maps.android:android-maps-utils:2.2.0"
 
     // GeoTools
-    implementation "org.geotools:gt-main:$project.geotoolsVersion"
+    implementation "org.locationtech.jts:jts-core:1.19.0"
+    implementation "org.locationtech.jts.io:jts-io-common:1.19.0"
+    //implementation "org.geotools:gt-geojson-core:$project.geotoolsVersion"
```

Comment:
Delete? Also, I think we can remove the repo URL in the other gradle file if we remove this.

Reply:
Ah yes, I missed this one!